### ISSUE TITLE
Views not working

### ISSUE THREAD

**Issue by andruli** [simolus3/moor#1280]
**Describe the bug**

Importing a view on a `.moor` file breaks the code generation.

**Error**
```
$ flutter packages pub run build_runner build
...
[SEVERE] moor_generator:moor_generator on lib/db/db.dart (cached):

NoSuchMethodError: The getter 'references' was called on null.
Receiver: null
Tried calling: references
[SEVERE] Failed after 70ms
```

**How to replicate**
I created a minimal repo that breaks the build -> https://github.com/andruli/moor-views

It has:
- A `Persons` class
- A `views.moor` file that creates a view
- A `query.moor` file that attemts to use the view

**Versions**
I'm using the latest available versions of flutter and moor.

<details>
  <summary>Flutter doctor</summary>
  
  ```
flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.2.2, on macOS 11.3.1 20E241 darwin-x64, locale en-AR)
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 3.5)
[✓] VS Code (version 1.57.1)
[✓] Connected device (1 available)

• No issues found!
```
</details>

<details>
  <summary>Package versions</summary>
  ```
moor: 4.3.2
moor_generator: 4.3.1
sqlite3_flutter_libs: 0.5.0
path_provider: 2.0.2
path: 1.8.0
```
</details>

PS. Sorry if this is not a supported feature yet.

**Comment by simolus3** [comment]
Thank you for the detailed issue with a repo! I was able to reproduce this locally and I'm working on a fix.

**Comment by simolus3** [comment]
This is fixed with 5ac485b55926a8d797de8e142418582feb9f270a.

**Comment by andruli** [comment]
Thank you @simolus3 !
